server/mongo/src/storage.ts

```diff
@@ -691,6 +726,22 @@ abstract class MongoAdapterBase implements DbAdapter {
           const coll = this.collection(domain)
           const mongoQuery = this.translateQuery(_class, query)
 
+          if (options?.limit === 1) {
+            // Skip sort/projection/etc.
```

Comment:
will it work properly what if we query first sorted by date? if `mongoQuery` does not contain sort here, then the result will be incorrect.

Reply:
Yes, make seance, will fix

Reply:
Resolved